SpywareFilter/sections/cookies.txt

```diff
@@ -10,6 +10,9 @@
 !------------------------------
 ! NOTE: Cookies specific
 !
+! ingles.com/spanishdict.com - tracking
+||ingles.com^$cookie=logglytrackingsession
+||spanishdict.com^$cookie=logglytrackingsession
```

Comment:
Can be merged into one rule
```suggestion
$cookie=logglytrackingsession,domain=ingles.com|spanishdict.com
```
or can be changed to generic.
Description:
https://www.plotto.com/cookies